eventProcessor/app.js

```diff
@@ -105,10 +109,11 @@ export async function lambdaHandler(event) {
       isBase64Encoded: false,
     };
   } catch (err) {
+    err.statusCode ??= 500;
     logger.error('failed to process Xray event', { err });
     response = {
       statusCode: err.statusCode,
-      body: err,
+      body: err.toString(),
```

Comment:
why `.toString()`?

Reply:
We don't necessary want the entire `err` object to be part of the response. `toString()` ensures only the textual part is used. This also allows any custom `Error` type to customize their own `toString()` output.